tools/validators/instance_validator/tests/fake_instances/GOOD/links_passthrough.yaml

```diff
@@ -35,18 +35,18 @@ VIRTUAL-ENTITY-B-GUID:
 GATEWAY-ENTITY-GUID:
   type: GATEWAYS/PASSTHROUGH
   code: GATEWAY-ENTITY
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
done.